rascaline/src/calculators/lode/spherical_expansion.rs

```diff
@@ -22,21 +22,38 @@ use crate::math::{compute_k_vectors, KVector};
 use crate::math::{gamma_ui, gamma};
 
 
-/// TODO
+/// Parameters for LODE spherical expansion calculator.
+///
+/// The spherical expansion is at the core of representations in the LODE
+/// (long-distance equivariant) family. See [this
+/// article](https://aip.scitation.org/doi/10.1063/1.5128375) for more 
+/// information on the LODE representation, and how it is implemented in rascaline.
 #[derive(Debug, Clone)]
 #[derive(serde::Deserialize, serde::Serialize, schemars::JsonSchema)]
 pub struct LodeSphericalExpansionParameters {
-    /// Spherical cutoff to use for atomic environments
+    /// Spherical realspace cutoff to use for atomic environments.
+    /// Note that this cutoff is only used for the projection of the density. 
+    /// In contrast to SOAP, LODE also takes atoms outside of this cutoff into 
+    /// account for the density.
     pub cutoff: f64,
+    /// Spherical reciprocal cutoff. If 0 k_cutoff
+    /// is set to 1.2 π / atomic_gaussian_width, which is reasonable value
+    /// for most systems.
+    pub k_cutoff: f64,
```

Comment:
To make the intent clearer, we can use an Option here, and maybe make this hyper optional (see how we handle radial_scaling in SOAP)

```suggestion
    pub k_cutoff: Option<f64>,
```